src/buffer/out/textBuffer.cpp

```diff
@@ -779,10 +847,6 @@ void TextBuffer::ScrollRows(const til::CoordType firstRow, const til::CoordType
         // - end
         std::rotate(_storage.begin() + firstRow, _storage.begin() + firstRow + size, _storage.begin() + firstRow + size + delta);
     }
-
-    // Renumber the IDs now that we've rearranged where the rows sit within the buffer.
-    // Refreshing should also delegate to the UnicodeStorage to re-key all the stored unicode sequences (where applicable).
-    _RefreshRowIDs(std::nullopt);

```

Comment:
(Note to self)
What were the row IDs used for beforehand? I should take a look at that haha.

Reply:
Row IDs were an utter atrocity